all.js

```diff
@@ -8,6 +8,17 @@ var simssh = require('simple-ssh');
 var scp2 = require('scp2')
 var biHelper = require('./biHelper.js');
 
+// [REVIEW] we should have only one uploadFiles, either uploadFiles, or uploadFilesViaScp
```

Comment:
yes, that's what i was thinking, uploadFiles is only used in raspberry pi c, and I can just switch to SCP as well